src/memory-analyzer/Makefile

```diff
@@ -0,0 +1,37 @@
+SRC = analyze_symbol.cpp\
+			gdb_api.cpp \
+			memory_analyzer_main.cpp \
+			memory_analyzer_parse_options.cpp
+      # Empty last line
+
+INCLUDES= -I ..
+
+LIBS = \
+	../ansi-c/ansi-c.a \
```

Comment:
Do not hard-code `.a`, use `$(LIBEXT)` as done elsewhere - this is currently causing build failures on Windows.